src/Spryker/Service/OtelConsoleInstrumentation/OpenTelemetry/ConsoleInstrumentation.php

```diff
@@ -97,14 +111,71 @@ function: static::METHOD_NAME,
                 }
 
                 $span = static::handleError($scope);
-                SamplerSpanFilter::filter($span, true);
+                $span->setAttribute('hasCustomParams', true);
+                $span->setAttributes(CustomParameterStorage::getInstance()->getAttributes());
+                $span = SamplerSpanFilter::filter($span, true);
 
                 $span->end();
             },
         );
         // phpcs:enable
     }
 
+    /**
+     * @param string $className
+     * @param string $application
+     * @param \Spryker\Shared\Opentelemetry\Request\RequestProcessor $requestProcessor
+     *
+     * @return void
+     */
+    protected static function registerBootstrapHook(string $className, string $application, RequestProcessor $requestProcessor): void
+    {
+        hook(
+            class: $className,
+            function: '__construct',
+            pre: static function ($instance, array $params, string $class, string $function, ?string $filename, ?int $lineno) use ($requestProcessor, $application): void {
+                putenv('OTEL_SERVICE_NAME=' . $application);
+                $instrumentation = CachedInstrumentation::getCachedInstrumentation();
+                if ($instrumentation === null || $requestProcessor->getRequest() === null) {
+                    return;
+                }
+
+                if (!defined('OTEL_CLI_TRACE_ID')) {
+                    define('OTEL_CLI_TRACE_ID', uuid_create());
+                }
+
+                $input = [static::CLI_TRACE_ID => OTEL_CLI_TRACE_ID];
+                TraceContextPropagator::getInstance()->inject($input);
+
+                $span = $instrumentation
+                    ->tracer()
+                    ->spanBuilder(static::formatSpanName($requestProcessor->getRequest()))
```

Comment:
Probably make it the same as in `register()` 
```suggestion
                    ->spanBuilder('Run: '  . static::formatSpanName($requestProcessor->getRequest()))
```